kombu/transport/mongodb.py

```diff
@@ -313,6 +313,7 @@ def _parse_uri(self, scheme='mongodb://'):
                                  if self.connect_timeout else None),
         }
         options.update(parsed['options'])
+        options.update(client.transport_options)
```

Comment:
can you please add test for the proposed change?

Reply:
OK, i will add test later.

Reply:
can you address the tests please?